### ISSUE TITLE
Texture confusion

### ISSUE THREAD

**Issue by Qfather** [roro08/Sketchfab-Ripper#13]
hope it resolves Or there is a local image restoration solution

**Comment by adamdived** [comment]
Yeah not working anymore with textures. I think that those maps you download are the good ones, but get encoded on the server side and decoded back to normal by the 3d player.

**Comment by sajjadtiger** [comment]
textures are encoded 
 solution?

**Comment by jack197553** [comment]
![banners_monza d3d4c79cbb134da8aeca85c4258c09ed](https://user-images.githubusercontent.com/53310265/166136591-5bf8f13a-376a-4fbe-b8e1-f0f3976418aa.png)
